app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
why was this deleted here? I assume a change this drastic would be included in another PR followed (likely) by a breaking api change as this is supported in ember-redux 1x right now